addons/web_editor/static/src/xml/editor.xml

```diff
@@ -12,35 +12,51 @@
                     data-bs-toggle="dropdown" data-bs-original-title="Text style" tabindex="-1" aria-expanded="false">
                     <span title="Text style">Normal</span>
                 </button>
-                <ul class="dropdown-menu">
-                    <li id="paragraph-dropdown-item">
-                        <a class="dropdown-item" href="#" id="paragraph" data-call="setTag" data-arg1="p">Normal</a>
+                <ul class="dropdown-menu" style="max-height:50vh;overflow-y:auto">
```

Comment:
Ok @qsm-odoo my solution was a bit ugly but we have to do something because now the dropdowns can overflow... Do you want me to compute the height of the dropdown when the user open it ?

![image](https://github.com/odoo/odoo/assets/78849981/851fb943-9eaf-4b4c-9db6-9d732fa336c2)

Reply:
To confirm with the related POs but I think that in the backend, we could just not show Header 1 Display X. Maybe even not all the header X. If not that, then yes, I guess you can restore what you did but for the backend only.